cmd/mdmctl/mdmcert.download.go

```diff
@@ -213,7 +219,10 @@ func (cmd *mdmcertDownloadCommand) Run(args []string) error {
 		if err != nil {
 			return errors.Wrap(err, "reading PKI private key")
 		}
-		ioutil.WriteFile("/tmp/fubar.p7", pkcsBytes, 0666)
+		err = ioutil.WriteFile("/tmp/fubar.p7", pkcsBytes, 0666)
```

Comment:
This looks like never-removed debugging code. Not sure why we'd need to write this out.

Reply:
I doubt we do :), especially with a name like "fubar"